### ISSUE TITLE
small error in exp51.py

### ISSUE THREAD

**Issue by murphyk** [zalandoresearch/CRISP#3]
https://github.com/zalandoresearch/CRISP/blob/master/code/experiments/exp_5.1/exp51.py#L19
The code is currently:
```
contacts_3 = init_contacts(S=S, T=T, R0=R0_3, seed=args.seed )
fig_3 = make_figure(**contacts_2**, t_branch=120, contacts_branch=contacts_3)
```
However, the first argument should be contacts_3.

**Comment by rvollgraf** [comment]
No, that's actually correct. The code produces Figure 2 (bottom-right), the first argument is for the solid lines and and contacts_branch=contacts_3 creates the dashed lines that represent the release after lockdown
